website/src/theme/DocPage/index.js

```diff
@@ -50,16 +50,34 @@ function DocPageContent({
 
   // Check if page available for current version
   const { versionedPages } = usePluginData('docusaurus-build-global-data-plugin');
-  const { version: dbtVersion, EOLDate, latestStableRelease } = useContext(VersionContext)
+  const { version: dbtVersion, EOLDate, isPrerelease, latestStableRelease } = useContext(VersionContext)
   const { pageAvailable, firstAvailableVersion } = pageVersionCheck(dbtVersion, versionedPages, currentDocRoute.path)
 
-  // Check End of Life date and show unsupported banner if depricated version
+  // Check whether this version is a isPrerelease, and show banner if so
+  const [PreData, setPreData] = useState({
+    showisPrereleaseBanner: false,
+    isPrereleaseBannerText: ''
+  })
+
+  // Check End of Life date and show unsupported banner if deprecated version
   const [EOLData, setEOLData] = useState({
     showEOLBanner: false,
     EOLBannerText: ''
   })
 
   useEffect(() => {
+    // If version is not isPrerelease, do not show banner
+    if(!isPrerelease) {
+      setPreData({
+        showisPrereleaseBanner: false,
+        isPrereleaseBannerText: ''
+      })
+    } else {
+        setPreData({
+          showisPrereleaseBanner: true,
+          isPrereleaseBannerText  : `This is a prerelease version. The latest stable version is ${latestStableRelease}`
```

Comment:
@JKarlavige or @jtcohen6 How can we programmatically add the prerelease version here? 
```suggestion
          isPrereleaseBannerText  : `You are currently viewing v1.1, which is a prerelease of dbt Core. The latest stable version is ${latestStableRelease}`
```

Reply:
@runleonarun I think I can do this! comma ça ?

<img width="1295" alt="Screenshot 2022-03-29 at 20 38 01" src="https://user-images.githubusercontent.com/13897643/160681871-98f878c2-d2cf-4037-9f44-be5dbd62f58c.png">